arangod/Pregel/Conductor.h

```diff
@@ -133,7 +133,8 @@ class Conductor {
             std::vector<CollectionID> const& vertexCollections,
```

Comment:
since you've started using `shared_from_this()` please, make `Conductor` constructor private, introduce a factory method returning `std::shared_ptr<Conductor>` and leave corresponding comment.

Reply:
Unfortunately when making the constructor private we can no longer use `make_shared`.

Reply:
You could use this workaround: https://godbolt.org/z/K85cv1xGq